Makefile

```diff
@@ -6,7 +6,7 @@ all: deps lint man package install
 
 # Install dependent pip packages, needed to lint or build
 deps:
-	$(PYTHON) -m pip install black isort pylint mypy build twine pytest pytest-mock
+	$(PYTHON) -m pip install black==24.1.1 isort pylint mypy build twine pytest pytest-mock
 	$(PYTHON) -m pip install .
 
```

Comment:
Is there a reason not to pin this version in `setup.cfg`, and have `make deps` just do `$(PYTHON) -m pip install .[dev]`?

Reply:
It seems like even if this doesn't do `pip install .[dev]` we should maybe also pin in `setup.cfg`?

Reply:
whoa TIL